src/Core/Content/ImportExport/Strategy/Import/BatchImportStrategy.php

```diff
@@ -75,6 +75,7 @@ public function commit(Config $config, Progress $progress, Context $context): Im
                 $result = $this->repository->upsert($records, $context);
             }
 
+            // @deprecated tag:v6.7.0 - remove the whole loop with no replacement
```

Comment:
Things like this can also already be prepared by checking the major feature flag with an if, which we can remove next week with all the other deprecations.

Benefit of that approach: If you activate the feature flag now, you shouldn't get any deprecation warnings because that event class should already not be used anymore